test/mocker/rtm.spec.js

```diff
@@ -33,10 +33,13 @@ describe('mocker: rtm', function () {
       Server: sinon.stub()
     }
 
-    serverMock = 'server'
+    serverMock = {
+      close: sinon.stub().returns()
```

Comment:
let's add some assertions to make sure this is called correctly. You'll need to use `yields` since it is an async function invoking a callback.